Code/Editor/CryEdit.h

```diff
@@ -113,6 +113,18 @@ AZ_POP_DISABLE_DLL_EXPORT_BASECLASS_WARNING
     bool CreateLevel(bool& wasCreateLevelOperationCancelled);
     void LoadFile(QString fileName);
     void ForceNextIdleProcessing() { m_bForceProcessIdle = true; }
+#if defined(CARBONATED)
+    bool ToggleForceInactive() { m_bForcePause = !m_bForcePause; return m_bForcePause; }
+    bool SetAIPause(bool paused);
+    bool SetAICover(bool show);
+    int SetCoverElev(int elev);
+    int SetTimeScale(int inverseScale);
+    int SetAIImap(int imap);
+    int SetAIImapSquad(int squad);
+    int SetAIDebugSquad(int squad);
+    int SetAIDebugChar(int chararacter);
+    int SetAIInputOverride(int team);
+#endif
```

Comment:
Same as above.

Reply:
It was an original Carbonated patch in LY engine.
But it really looks like a proxy.
Moved this functionality into PR: https://github.com/carbonated-dev/o3de-gruber/pull/565

Reply:
awesome, thank you.  I recall that it came from original LY engine code, there's a longstanding ticket to move it out that we never got to.  Tackling it now is a good time :)